src/Modules/StringExtensions.bas

```diff
@@ -3,8 +3,10 @@ Public Function TrimLeft(text As String, c As String) As String
     Dim textLength As Long
     textLength = Len(text)
     
+    Dim firstCharacter As String
+    
     Do While textLength > 0
-        Dim firstCharacter As String
+       

```

Comment:
this new line looks a bit weird. could you please remove it?

Reply:
Do you prefer multiple variable declarations inside a loop?

Reply:
Nope. But removing a blank line doesn't add a new declaration 🙃